File: app/src/main/java/com/duncancodes/buttoncounter/MainActivity.java
package com.duncancodes.buttoncounter;

import android.content.Intent;
import android.support.annotation.NonNull;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.text.method.ScrollingMovementMethod;
import android.util.Log;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.ImageView;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {
    private static final String TAG = MainActivity.class.getName();
    private EditText userInput; //storing EditText widget data in variable userInput
    private TextView textView; //as above etc
    private final String TEXT_CONTENTS = "";

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        Log.d(TAG, "onCreate: in");
        setContentView(R.layout.activity_main);

        ImageView ivNotepad = (ImageView)findViewById(R.id.imageViewNotepad);
        ivNotepad.setOnClickListener(v -> {
            Log.d(TAG, "Image view clicked");

            Intent intent = new Intent(MainActivity.this, NotepadActivity.class);
            startActivity(intent);
        });
    }


}

File: app/src/main/java/com/duncancodes/buttoncounter/MainActivity.kt
package com.duncancodes.buttoncounter

import android.support.v7.app.AppCompatActivity
import android.widget.EditText
import android.widget.TextView
import android.os.Bundle
import com.duncancodes.buttoncounter.ButtonCounterActivity
import com.duncancodes.buttoncounter.R
import android.text.method.ScrollingMovementMethod
import com.duncancodes.buttoncounter.MainActivity
import android.content.Intent
import android.util.Log
import android.view.View
import android.widget.ImageView
import com.duncancodes.buttoncounter.NotepadActivity

class MainActivity : AppCompatActivity() {
    private val userInput : EditText? = null //storing EditText widget data in variable userInput
    private val textView : TextView? = null //as above etc
    private val TEXT_CONTENTS = ""
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        Log.d(TAG, "onCreate: in")
        setContentView(R.layout.activity_main)
        val ivNotepad = findViewById<View>(R.id.imageViewNotepad) as ImageView
        ivNotepad.setOnClickListener { v: View? ->
            Log.d(TAG, "Image view clicked")
            val intent = Intent(this@MainActivity, NotepadActivity::class.java)
            startActivity(intent)
        }
    }

    companion object {
        private val TAG = MainActivity::class.java.name
    }
}